<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_1">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item alignment="Qt::AlignLeft">
        <widget class="QTableWidget" name="twAssociation">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhFormattedNumbersOnly</set>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <property name="rowCount">
          <number>64</number>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>102</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>30</number>
         </attribute>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <column>
          <property name="text">
           <string>value</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>new value</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QComboBox" name="cmbBox"/>
         </item>
         <item>
          <widget class="QPushButton" name="pbApply">
           <property name="maximumSize">
            <size>
             <width>113</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>&lt;- Apply Csv Map</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="leText">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
